src/mentorpal/classifiers/arch/lstm_v2/training/__init__.py

```diff
@@ -68,144 +91,117 @@ def __load_training_data(self, path):
-
         #for each data point, get w2v vector for the question and store in train_vectors.
-        #instance=<question, processed_question, topic, answer_id, answer_text>
+        #instance=<question, processed_question, topic, answer_id, answer_text, univariate_features[5 to 12]>
         for instance in train_data:
-            w2v_vector, lstm_vector=self._LSTMClassifier__get_w2v(instance[1])
-            # train_vectors.append([instance[0],w2v_vector.tolist(),instance[2],instance[4]])
-            train_vectors.append([instance[0],w2v_vector.tolist(),instance[2],instance[3],instance[4],instance[5],instance[6],instance[7],instance[8],instance[9],instance[10],instance[11],instance[12]])
+            w2v_vector, lstm_vector = self.w2v.w2v_for_question(instance[1])
+            train_vectors.append([instance[0],w2v_vector.tolist(),instance[2],instance[4],instance[5:13]])
             lstm_train_vectors.append(lstm_vector)
-
         #For the LSTM, each training sample will have a max dimension of 300 x 25. For those that don't, the pad_sequences
         #function will pad sequences of [0, 0, 0, 0....] vectors to the end of each sample.
         padded_vectors=pad_sequences(lstm_train_vectors,maxlen=25, dtype='float32',padding='post',truncating='post',value=0.)
         lstm_train_vectors=padded_vectors
-
         return train_vectors, lstm_train_vectors
 
+
     def __load_topic_vectors(self, train_vectors, lstm_train_vectors):
         lstm_train_data = []
-
         #Generate the sparse topic train_vectors
         for i in range(len(train_vectors)):
             question=train_vectors[i][0]
-            vector=train_vectors[i][1]
             current_topics=train_vectors[i][2]
             topic_vector=[0]*len(self.mentor.topics)
             for j in range(len(self.mentor.topics)):
                 if self.mentor.topics[j] in current_topics:
                     topic_vector[j]=1
             train_vectors[i][2]=topic_vector
             lstm_train_data.append([question, lstm_train_vectors[i].tolist(), topic_vector])
-
         return train_vectors, lstm_train_data
 
+
     def __load_xy_train(self, lstm_train_data):
         x_train = None
         y_train = None
-
         x_train=[lstm_train_data[i][1] for i in range(len(lstm_train_data))] #no of utterances * no_of_sequences * 300
         y_train=[lstm_train_data[i][2] for i in range(len(lstm_train_data))] #No_of_utterances * no_of_classes (40)
         x_train=np.asarray(x_train)
-
         return x_train, y_train
 
-    def __load_fused(self, train_data, train_topic_vectors):
+
+    def __load_fused_unfused(self, train_data, train_topic_vectors):
         x_train_fused=[]
         y_train_fused=[]
+        x_train_unfused=[]
+        y_train_unfused=[]
+
+        x_train_unfused=[train_data[i][1] for i in range(len(train_data))]
+        y_train_unfused=[train_data[i][3] for i in range(len(train_data))]
+        x_train_unfused=np.asarray(x_train_unfused)
```

Comment:
similar to comment above, this should be a single line, e.g.

`x_train_unfused=np.asarray([train_data[i][3] for i in range(len(train_data))])`

...same for all similar vars in this block